test/builders/models/UserBuilder.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package builders.models
+
+import builders.models.mongo.EmploymentUserDataBuilder.anEmploymentUserData
+import models.User
+import play.api.mvc.AnyContentAsEmpty
+import play.api.test.FakeRequest
+
+object UserBuilder {
+  val aUserRequest: User[AnyContentAsEmpty.type] = User(
+    mtditid = anEmploymentUserData.mtdItId,
+    arn = None,
+    nino = anEmploymentUserData.nino,
+    sessionId = anEmploymentUserData.sessionId,
+    affinityGroup = "affinityGroup")(FakeRequest())
```

Comment:
does this need to be set to "affinityGroup" or could it reference the affinityGroup variable within IntegrationTest?

Reply:
It can't. Builders are in 'test' and cannot see what is in 'it'.

Reply:
okay cool